docs/api.rst

```diff
@@ -0,0 +1,19 @@
+PyLmod API Docs
+---------------
+
+For convenient reference in development, here are the PyLmod
+API docs.
+
+Library
+=======
+
+API documentation
+
+.. automodule:: pylmod
```

Comment:
I know why I was thinking TOC would work, I didn't notice we are auto loading the whole library.  It might be worth laying down each class we want to document.  Something like:

```
GradeBook Class
~~~~~~~~~~~~~
.. automodule:: pylmod.GradeBook
    :members:
    :undoc-members:
    :show-inheritance:

Membership Class
~~~~~~~~~~~~~
.. automodule:: pylmod.Membership
    :members:
    :undoc-members:
    :show-inheritance:
```

Totally your call though, that approach means we have to manually add new classes to document, but does make navigation between the classes work.